### PR TITLE
Update setup instructions in README

### PR DESCRIPTION
**Pull request by ErinHales**
## Motivation

Updated root readme with all the setup instructions needed to get the project running

## Checklist for your Pull Request (PR)
  - [ ] I added an entry to `CHANGELOG.md` with this PR
  - [ ] If I added new functionality, I added tests covering it.
  - [ ] If I fixed a bug, I added a regression test to prevent the bug from silently reappearing again.
  - [ ] I checked whether I should update the docs and did so by submitting a PR to https://github.com/blockscout/docs
  - [ ] If I added/changed/removed ENV var, I submitted a PR to https://github.com/blockscout/docs to update the list of env vars at https://github.com/blockscout/docs/blob/master/for-developers/information-and-settings/env-variables.md and I updated the version to `master` in the Version column. Changes will be reflected in this table: https://docs.blockscout.com/for-developers/information-and-settings/env-variables. 
  - [ ] If I add new indices into DB, I checked, that they are not redundant with PGHero or other tools

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,11 +33,85 @@ BlockScout supports a number of projects. Hosted instances include POA Network,
 - [List of hosted mainnets, testnets, and additional chains using BlockScout](https://docs.blockscout.com/for-projects/supported-projects)
 - [Hosted instance versions](https://docs.blockscout.com/about/use-cases/hosted-blockscout)
 
-
 ## Getting Started
 
+1. Install requirements
+
+    For a complete list of requirements, see the [blockscout docs](https://docs.blockscout.com/for-developers/information-and-settings/requirements).
```
Comment (aaronmgdr) [inline README.md:40]: to get elixir command i had to modify my PATH in my zsh file 

i did `asdf which elixir` cut off the end and then copied that to zshrc like PATH=/$HOME/asdf-something/.bin:$PATH

File: README.md [hunk 2]
```diff
@@ -33,11 +33,85 @@ BlockScout supports a number of projects. Hosted instances include POA Network,
...
+
+4. Install Deps and Compile
+
+    ```shell
+    mix local.hex --force
+    mix local.rebar --force
+    mix deps.get
+    cd apps/block_scout_web/assets/ && \
+      npm install && \
+      npm run deploy && \
```
Comment (aaronmgdr) [inline README.md:60]: should this be npm run deploy?

Comment (aaronmgdr) [reply README.md:60]: deploy sounds like production

File: README.md [hunk 3]
```diff
@@ -33,11 +33,85 @@ BlockScout supports a number of projects. Hosted instances include POA Network,
...
+      cd -
+    cd apps/explorer/ && \
+      npm install && \
```
Comment (aaronmgdr) [inline README.md:63]: can we do `yarn` instead of `npm`  ?

**Comment by aaronmgdr** [comment]
might help to add a TROUBLESHOOTing file or part of readme. 

one thing i just did was deleted the `_build` and `deps` folders and reinstalled.

File: README.md [hunk 4]
```diff
@@ -33,11 +33,85 @@ BlockScout supports a number of projects. Hosted instances include POA Network,
...
+      cd -
+    mix compile
+    ```
+
+5. Start blockscout in a docker container
```
Comment (aaronmgdr) [inline README.md:68]: thoughts on adding a 

"if you dont have docker installed get it [here](dockerlink)" 

Part?

Comment (ErinHales) [reply README.md:70]: Added!

File: README.md (hunk 2 above)
Comment (ErinHales) [reply README.md]: You're right, this should be build

File: README.md [hunk 5]
```diff
@@ -33,16 +33,99 @@ BlockScout supports a number of projects. Hosted instances include POA Network,
...
+     > If it is still not recognizing Elixir, you can try adding this to your `~/.zshrc` file: `PATH=/$HOME/asdf/installs/elixir/1.9.1/.bin:$PATH`.  You can find your correct path using `asdf which elixir`.
+
+2. Set up some default configuration
+
+    ```shell
+    cp apps/explorer/config/dev.secret.exs.example apps/explorer/config/dev.secret.exs
+    cp apps/block_scout_web/config/dev.secret.exs.example apps/block_scout_web/config/dev.secret.exs
+    ```
+
+3. Add Env Variables
```
Comment (carterqw2) [inline README.md:52]: Maybe that could be done when you start docker (see below).

File: README.md [hunk 6]
```diff
@@ -33,16 +33,99 @@ BlockScout supports a number of projects. Hosted instances include POA Network,
...
+
+5. Start blockscout in a docker container
+
+     > If you do not already have docker installed, you can get it here:  
+     >   - [Mac](https://hub.docker.com/editions/community/docker-ce-desktop-mac)
+     >   - [Windows](https://hub.docker.com/editions/community/docker-ce-desktop-windows)
+
+    ```shell
+    cd docker
+    make start
```
Comment (carterqw2) [inline README.md:78]: Something like
```suggestion
NETWORK=Celo ETHEREUM_JSONRPC_VARIANT=geth ETHEREUM_JSONRPC_HTTP_URL=http://104.198.100.15:8545 ETHEREUM_JSONRPC_WS_URL=ws://104.198.100.15:8546 COIN=cGLD make start
```

should take care of everything including creating the database container, running migrations and starting indexer, so steps 6-9 are not needed really. Maybe it worth mentioning that values for 
`ETHEREUM_JSONRPC_HTTP_URL` and `ETHEREUM_JSONRPC_WS_URL` may vary and should point to a running archive node.

**Review by carterqw2 — CHANGES_REQUESTED** [review]

File: README.md [hunk 7]
```diff
@@ -110,8 +110,8 @@ BlockScout supports a number of projects. Hosted instances include POA Network,
 
     ```shell
     cd docker
-    NETWORK=Celo ETHEREUM_JSONRPC_VARIANT=geth
-    ETHEREUM_JSONRPC_HTTP_URL=http://104.198.100.15:8545 ETHEREUM_JSONRPC_WS_URL=ws://104.198.100.15:8546 COIN=cGLD
+    export NETWORK=Celo ETHEREUM_JSONRPC_VARIANT=geth
```
Comment (carterqw2) [inline README.md:113]: Actually, `export` shouldn't be needed in this case as that's the standard way of passing parameters to `make`.
```
make NETWORK=Celo ETHEREUM_JSONRPC_VARIANT=geth ETHEREUM_JSONRPC_HTTP_URL=http://104.198.100.15:8545 ETHEREUM_JSONRPC_WS_URL=ws://104.198.100.15:8546 COIN=cGLD start
```
should also work.

**Review by carterqw2 — APPROVED** [review]

**Pull request merged**